gitops/snapshot.go

```diff
@@ -27,6 +27,9 @@ const (
 	// BuildPipelineRunFinishTimeLabel contains the build PipelineRun finish time of the Snapshot.
 	BuildPipelineRunFinishTimeLabel = "test.appstudio.openshift.io/pipelinerunfinishtime"
 
+	// BuildPipelineRunNameLabel contains the build PipelineRun name
+	BuildPipelineRunNameLabel = "test.appstudio.openshift.io/build-pipelinerun"
```

Comment:
TBD: this needs decision about proper name

Reply:
I'm leaning towards having it as `appstudio.openshift.io/build-pipelinerun` as I could see this getting used by the UI or another service in the future.